11lesson_dz.py

```diff
@@ -0,0 +1,45 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+
+import time
+
+def calculate_time(func):
+    def inner(*args, **kwargs):
+        begin = time.strftime("%H:%M:%S")
+        func(*args, **kwargs)
```

Comment:
It is better to use `return` statement here to make it works with functions that return some value

Reply:
Could you please specify where exactly is it better to use "return" statement? I do have some difficulties with functions and try working on them

Reply:
Right in this line: `return func(*args, **kwargs)`